test/extensions/filters/http/ext_proc/unit_test_fuzz/ext_proc_unit_test_fuzz.cc

```diff
@@ -30,6 +30,12 @@ class FuzzerMocks {
     ON_CALL(encoder_callbacks_, encodingBuffer()).WillByDefault(Return(&buffer_));
     ON_CALL(decoder_callbacks_, decoderBufferLimit()).WillByDefault(Return(1024));
     ON_CALL(encoder_callbacks_, encoderBufferLimit()).WillByDefault(Return(1024));
+    ON_CALL(decoder_callbacks_, injectDecodedDataToFilterChain(_, _))
```

Comment:
qq: Is this something can be done in the centralized place /test/mocks/http/mocks.h? Other filter can run into same issue if have similar behavior?

Reply:
Very few filters call this inject data function.  So, keep it within individual filter which uses it make more sense to me.